lib/operations/create_collection.js

```diff
@@ -22,7 +22,7 @@ const illegalCommandFields = [
   'raw',
   'readPreference',
   'session',
-  'writeConcern'
```

Comment:
Does this mean we do not support writeConcern on `createCollection`?

Reply:
This means that `writeConcern` will be added to the command [here](https://github.com/mongodb/node-mongodb-native/pull/2026/files#diff-117da8126ba45d60347edb946fd27a47L46). Looking at this more, it seems like we should keep `writeConcern` in `illegalCommandFields` because it should be specified as an option, but not added to the command. I'm actually not sure why what the "optional parameters" referenced [here](https://github.com/mongodb/node-mongodb-native/pull/2026/files#diff-117da8126ba45d60347edb946fd27a47L41) are.

Reply:
@rosemaryy I think we should add `'writeConcern'` back in here. Sorry for telling you to remove it!